README.md

```diff
@@ -1,7 +1,112 @@
-marta-python
-============
+# marta-python
 
 Python library for accessing MARTA real-time API
 
-* [getBuses([route#])](https://github.com/itsmarta/marta-python/blob/master/getVehicles.py#L10)
-* [getTrains([station_name])](https://github.com/itsmarta/marta-python/blob/master/getVehicles.py#L40)
\ No newline at end of file
+## Testing
+
+```
+# Install tox
+pip install tox
+# Run tests
+tox
+```
+
+## Installing
+
+```
+python setup.py install
```

Comment:
I'm not really a python developer (so I don't know all of the conventions), but should the README mention that you need python 3? I haven't run python code in a while and got stuck on an error importing `JSONDecodeError` when using 2.7

Reply:
This library should be compatible with 2.7 and 3.x. `JSONDecodeError` wasn't used anyway, so I removed it and the tests pass against 2.7 and 3.7. Nice catch!